02-peer-protocol.md

```diff
@@ -633,19 +673,20 @@ There are four values that need be derived:
 1. The `cltv_expiry_delta` for channels, `3R+2G+2S`: if in doubt, a
    `cltv_expiry_delta` of 12 is reasonable (R=2, G=1, S=2).
 
-2. For sent HTLCs: the timeout deadline after which the channel has to be failed
+2. The `cltv_expiry_delta` for sent HTLCs: the timeout deadline after which the channel has to be failed
```

Comment:
No, `cltv_expiry_delta` is an explicit field we advertise with channels, calculated at point one.

This would be:

2. The deadline for offered HTLCs: the timeout after which the channel has top be failed...

Reply:
Thanks much, Rusty, especially for these paired fixed. This was a case of me trying to clarify, but not being 100% sure on the correct answer. I'd usually query about these in advance, but I've found the work-hour difference a little challenging, so I let you see it in diffs instead. If this was a good way to find clarity for this sort of issue, I'll continue with it; but if you would have preferred discussion in advance, let me know, and we'll just do the back and forth first.

Reply:
No, this workflow is optimal!  You can also ask for clarifications in the github PR description, which doesn't show up in the final result but gives me a heads-up.